src/libraries/System.Threading/tests/EventWaitHandleTests.cs

```diff
@@ -158,8 +163,11 @@ public void OpenExisting_UnavailableName_Windows()
         {
             string name = Guid.NewGuid().ToString("N");
             Assert.Throws<WaitHandleCannotBeOpenedException>(() => EventWaitHandle.OpenExisting(name));
-            EventWaitHandle ignored;
-            Assert.False(EventWaitHandle.TryOpenExisting(name, out ignored));
+            Assert.False(EventWaitHandle.TryOpenExisting(name, out _));
```

Comment:
Nit: can/should we add an assert for the value of the out parameter?  Same with below.